docs/quantum/tutorials/beginner_tutorial.rst

```diff
@@ -1,3 +1,8 @@
+<<<<<<< HEAD:docs/quantum/tutorials/beginner_tutorial.rst
```

Comment:
Merge conflict text here

Reply:
fixed in f3a1ab0c52451420608cef8255bb713f9634d53b